pp/integrations/plugin-packs/procedures/network-firewalls-juniper-ssg-snmp.md

```diff
@@ -2,36 +2,627 @@
+| Traffic-Generic-Name | Net-JuniperSSG-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
+| Traffic-Global       | Net-JuniperSSG-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each checks the name of the interface will appear    |            |
+| Vpn-Status           | Net-JuniperSSG-Vpn-Status-custom           | Check VPN tunnels status                                                                     | X          |
+| Vpn-Usage            | Net-JuniperSSG-Vpn-Usage-custom            | Check VPN tunnels inbound and outbound traffic                                               |            |
+
+> The services listed above are not created automatically when a host template is applied. To use them, [create a service manually](/docs/monitoring/basic-objects/services), then apply the service template you want.
+
+> If **Discovery** is checked, it means a service discovery rule exists for this service template.
+
+</TabItem>
+</Tabs>
+
+### Discovery rules
+
+#### Host discovery
+
+| Rule name       | Description                                                                                                                                                                                                                                  |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-SSG-custom** host template |
+
+More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).
+
+#### Service discovery
+
+| Rule name                | Description |
+|:-------------------------|:------------|
+| Net-Juniper-SSG-Vpn-Name |             |
+
+More information about discovering services automatically is available on the [dedicated page](/docs/monitoring/discovery/services-discovery)
+and in the [following chapter](/docs/monitoring/discovery/services-discovery/#discovery-rules).
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="Cpu" label="Cpu">
+
+| Metric name | Unit  |
+|:------------|:------|
+| cpu_1min    | %     |
+| cpu_5min    | %     |
+| cpu_15min   | %     |
+
+</TabItem>
+<TabItem value="Hardware" label="Hardware">
+
+Coming soon
```

Comment:
No metrics? 
If none, Explain that there are no metrics for this service template.